src/course-unit/header-navigations/HeaderNavigations.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+
+import messages from './messages';
+
+const HeaderNavigations = ({ headerNavigationsActions }) => {
+  const intl = useIntl();
+  const { handleViewLive, handlePreview } = headerNavigationsActions;
+
+  return (
+    <nav className="header-navigations ml-auto flex-shrink-0">
+      <Button
+        variant="outline-primary"
+        onClick={handleViewLive}
+      >
+        {intl.formatMessage(messages.viewLiveButton)}
+      </Button>
+      <Button
+        variant="outline-primary ml-2.5"
```

Comment:
[important] This margin adds unwanted space between buttons (correct 12 px). Let's remove it.

![image](https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/e11c795d-d03f-44cc-9b55-68541f2fb652)

Reply:
Fixed